__tests__/api.test.js

```diff
@@ -0,0 +1,25 @@
+const app = require("../app");
+const request = require("supertest");
+const data = require("../db/data/test-data/index");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+require("jest-sorted");
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe("GET /api/topics", () => {
+  test("should respond with a 200 status and an array containing all topics objs", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((data) => {
+        const output = data.body.topics;
+        expect(typeof output).toBe("object");
+        output.forEach((topic) => {
```

Comment:
we need test that does length check if going to through each object.  this forEach won't run on an empty array so you can get a false passing test!